### PR TITLE
Fixes #29352 - Create action_buttons in react

### PR DESCRIPTION
**Pull request by MariaAga**
A component to group one or more buttons into a drop down like Rubys action_buttons.
gets an array or buttons and returns a button or a dropdown of buttons depending on the number of buttons

### CODE REVIEW

**Comment by xprazak2** [comment]
Works for me, just when clicking the sedcond button with `'data-method': 'put'`, it generates post request.

Or does that happen just in storybook?

put-vs-post

**Comment by xprazak2** [comment]
Also needs a snapshot refresh.

**Comment by MariaAga** [comment]
Updates snapshots thanks!
About the post request:
In the Form Data it sends `_method=put` in the storybook

File: webpack/assets/javascripts/react_app/components/common/ActionButtons/ActionButtons.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { SplitButton, MenuItem, Button } from 'patternfly-react';
+
+/**
+ * Generate a button or a dropdown of buttons
+ * @param  {String} title The title of the button for the title and text inside the button
+ * @param  {Object} action action to preform when the button is click can be href with data-method or Onclick
+ * @return {Function} button component or splitbutton component
+ */
+export const ActionButtons = ({ buttons }) => {
+  if (!buttons.length) return '';
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/components/common/ActionButtons/ActionButtons.js]: nit: returning an empty string will return a text node for nothing, how about returning `null`?

Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/common/ActionButtons/ActionButtons.js]: Updated, thanks!

**Review by xprazak2 — APPROVED** [review]
I think we can merge unless there are other comments.

**Review by amirfefer — APPROVED** [review]
LGTM :+1: 
Thanks @MariaAga

**Pull request merged**